dev-packages/application-package/src/npm-registry.ts

```diff
@@ -49,6 +53,7 @@ export interface NodePackage {
     keywords?: string[];
     dependencies?: Dependencies;
     peerDependencies?: Dependencies;
+    peerDependenciesMeta?: PeerDependenciesMeta;
```

Comment:
I guess this just describes what we get from npm, right?

Reply:
Not npm, but the entry in the `package.json`. But yes, it's just part of the `package.json` structure.